src/ui/components/Icon/extensions-complex.svg

```diff
@@ -0,0 +1,165 @@
+
+<svg width="44px" height="48px" viewBox="0 0 44 48" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Presumably this replaces the fox icon everywhere? I think this one might need checking before we use this icon for the entire site.